### DISCUSSION TITLE
HSTS support

### DISCUSSION THREAD

**Discussion by hopeshared-lee** [dexidp/dex#3320] [Q&A]
Hi

I can't see "Strict-Transport-Security" was used in response header. Looks like DEX server won't return it.
Is it possible to add it into header? Or is there a way to add customized header in response?

Thanks!

**Comment by hopeshared-lee** [comment]
If it's not supported yet. Example code change can be found in https://github.com/golang/gddo/pull/562/files to add this header per request.

**Reply by nabokihms** [reply]
For now, Dex does not support configuring headers. Opened a PR with the implementation https://github.com/dexidp/dex/pull/3339
